docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -56,6 +60,11 @@ BC.OnSystemRequest in "Proprietary" Policy Table Update Flow
 ![Proprietary PTU](./assets/OnSystemRequest_in_Proprietary_PTU_flow.png)
 |||
 
+|||
+_EXTERNAL PROPRIETARY_ PTU flow
+![EXTERNAL proprietary](../PolicyUpdate/assets/diagram_PolicyUpdate_external_proprietary.png) 
```

Comment:
@DrachenkoAnastasiia Alright, so it looks like these links are case-sensitive, you need to use lower case letters in all of the directories (`../PolicyUpdate/...` should be `../policyupdate/...` in this case).